src/index.html

```diff
@@ -1,13 +1,586 @@
+                  <div class="cards__semicolon">
+                    <img
+                      src="images/testimonials/semicolon.svg"
+                      alt=""
+                      class="cards__semicolon-image"
+                    >
+                  </div>
+                  <div class="cards__text">
+                    AIR raises the agency bar to stratospheric heights on both creative output and client service.
+                  </div>
+                  <div class="cards__name">
+                    Michel Grover
+                  </div>
+                  <div class="cards__working">
+                    Hulu
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="page__section visionPassionResults">
+        <div class="visionPassionResults__content">
+          <h2 class="page__section-title visionPassionResults__title">
+            Vision, Passion, Results
+          </h2>
+          <div class="page__section-text visionPassionResults__text">
+            We are sure that first-rate job is possible only if all three components are united.
+          </div>
+          <button class="visionPassionResults__button">Apply</button>
+        </div>
+      </section>
+      <section class="page__section contact" id="contact-us">
+        <container class="contact__blocks">
+          <form action="">
+            <div class="contact__wrapper">
+              <div class="contact__form">
+                <h3 class="page__section-title contact__form-title">
+                  Send us a message
+                </h3>
+                <input
+                  required
+                  class="contact__form-input"
+                  type="email"
+                  name="email"
+                  placeholder="email@example.com"
+                >
+                <input
+                  required
+                  class="contact__form-input"
+                  type="text"
+                  name="name"
+                  placeholder="Your name"
+                >
+                <input
+                  required
+                  class="contact__form-input"
+                  type="textarea"
+                  name="message"
+                  placeholder="Your message"
+                >
+              </div>
+              <button type = "submit" class="contact__button contact__main-button">
+                Send
+              </button>
+            </div>
+          </form>
+          <div class="contact__contactUs">
+            <h3 class="page__section-title contact__contactUs-title">
+              Contact us
+            </h3>
+            <h4 class="contact__contactUs-subtitle">
+              Call us
+            </h4>
+            <a
+              class="contact__contactUs-text"
+              href="tel:654 321 987"
+              target="_blank"
+              rel="noopener noreferrer"
```

Comment:
remove this from phone link, it should initiate a call, but leave user on the same page

```suggestion
```